scipy/optimize/_linprog.py

```diff
@@ -338,13 +344,30 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     This section describes the available solvers that can be selected by the
     'method' parameter.
 
+    :ref:`'highs-simplex' <optimize.linprog-highs-simplex>` and
+    :ref:`'highs-ipm' <optimize.linprog-highs-ipm>` are interfaces to the
+    HiGHS simplex and interior-point method solvers [13]_, respectively.
+    'highs' chooses between the two automatically. These are the fastest linear
```

Comment:
Method `'highs'` is missing from the `method` parameter description - seems important to add.

Reply:
Added